schnorr_fun/src/frost.rs

```diff
@@ -276,26 +282,32 @@ impl core::fmt::Display for FinishKeyGenError {
 impl std::error::Error for FinishKeyGenError {}
 
 /// A joint FROST key
-#[derive(Clone, Debug, PartialEq)]
```

Comment:
This should be `PartialEq`. It's quite unfortunate that we can't derive it here.... I would make an exception to the rule about not putting type constraints on struct type params and put `FrostKey<T: PointType>` here.